ding/envs/env_manager/base_env_manager.py

```diff
@@ -130,7 +132,12 @@ def ready_obs(self) -> Dict[int, Any]:
             >>>     obs_dict = env_manager.ready_obs
             >>>     actions_dict = {env_id: model.forward(obs) for env_id, obs in obs_dict.items())}
         """
-        return {i: self._ready_obs[i] for i in range(self.env_num) if self._env_episode_count[i] < self._episode_num}
+        obs = [self._ready_obs[i] for i in range(self.env_num) if self._env_episode_count[i] < self._episode_num]
+        return tnp.stack(obs)
+
+    @property
+    def ready_obs_id(self) -> List[int]:
+        return [i for i in range(self.env_num) if self._env_episode_count[i] < self._episode_num]
```

Comment:
Add comment, and why episode count less than _episode_num are called ready obs?

Reply:
added some comments